src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export enum CHAIN_IDS {
+  'CANTO' = 7700,
+}
+export const CHAIN_ID = parseInt(
+  process.env.NEXT_PUBLIC_CHAIN_ID || CHAIN_IDS.CANTO.toString()
```

Comment:
...hmmmm. is it ok?

Reply:
I see no better option than fallback to some chain. 

we can simply explode the app in runtime to indicate that env is not configure correctly

Reply:
yeah we should throw an error if params are not set on app start

Reply:
there is a smart way: https://dev.to/sukruozdemir/how-to-validate-environment-file-in-nodejs-m2m but it probably an overkill for just two env vars